docs/undergrad.yaml

```diff
@@ -336,11 +336,11 @@ Single Variable Real Analysis:
     inverse trigonometric functions: 'real.arcsin'
     inverse hyperbolic trigonometric functions: 'real.arsinh'
   Integration:
-    integral over a segment of piecewise continuous functions: ''
+    integral over a segment of piecewise continuous functions: 'interval_integral'
     antiderivatives: 'https://en.wikipedia.org/wiki/Antiderivative'
-    Riemann sums: 'https://en.wikipedia.org/wiki/Riemann_sum'
+    Riemann sums: 'box_integral.integral'
```

Comment:
I wish we could have a more recognizable item here. Maybe [box_integral.integral_sum](https://leanprover-community.github.io/mathlib_docs/analysis/box_integral/basic.html#box_integral.integral_sum) is better.